backend/controller/observability/attributes.go

```diff
@@ -0,0 +1,5 @@
+package observability
+
+const (
+	fsmRefAttribute string = "ftl.fsm.ref"
```

Comment:
Given this is only used by `fsm.go` it could go there instead of this shared `attributes` file (unless there's plans to share it in the future).

Reply:
If we don't currently have any shared `attributes` then we can leave this file off for now and add it back if needed in the future.

Reply:
moved